File: euler001.py
for _ in range(int(input())):
    n, s = int(input()) - 1, 0
    s += 3*(n//3)*(n//3+1)
    s += 5*(n//5)*(n//5+1)
    s -= 15*(n//15)*(n//15+1)
    print(int(s//2)) 

File: Euler001.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    long long int t,n,sum;
    cin>>t;
    while(t--){
    cin>>n;
    sum=0;
    n--;
    sum += 3*(n/3)*(n/3+1);
    sum += 5*(n/5)*(n/5+1);
    sum -= 15*(n/15)*(n/15+1);
    cout<<sum/2<<endl;
        } 
    return 0;
}